services/wallet/following/manager.go

```diff
@@ -0,0 +1,96 @@
+package following
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/status-im/status-go/logutils"
+	"github.com/status-im/status-go/services/wallet/thirdparty/efp"
+)
+
+// Manager handles following address operations using EFP providers
+type Manager struct {
+	providers []efp.FollowingDataProvider
```

Comment:
we use a slice of providers for cases where multiple providers can be used to fetch the same type of data. I'm not super well versed in EFP, but it doesn't look like that's something we expect to happen, right?
Moreover, we're just using `m.providers[0]` in all methods, so making this a slice is misleading, better just make it `provider efp.FollowingDataProvider`